ocs_ci/utility/aws.py

```diff
@@ -304,18 +309,32 @@ def start_ec2_instances(self, instances, wait=False):
                 False otherwise
 
         """
+        stopping_instances = {
```

Comment:
This code block shouldn't be here.
This function should start ec2 instances and that's it - no need to check if the instances is in stopping state.
This sampling should happen, but not here